.gitignore

```diff
@@ -5,4 +5,5 @@ build
 .DS_STORE
 storybook-static
 .npmrc
-results
\ No newline at end of file
+results
```

Comment:
Waarom zit deze in de git ignore?

Reply:
ik weet niet of ik die results erin heb gestoken, ik kan het mij in ieder geval niet herinneren :-), het zit ook in de gitignore van de master zie ik